internal/schemautil/userconfig/handle.go

```diff
@@ -180,7 +180,7 @@ func handleArrayProperty(n string, p map[string]interface{}, t string) (map[stri
 
 		if an == "ip_filter" {
 			// TODO: Add ip_filter_object to this sanity check when DiffSuppressFunc is implemented for it.
-			s[jen.Id("DiffSuppressFunc")] = jen.Qual(SchemaUtilPackage, "IpFilterArrayDiffSuppressFunc")
+			s[jen.Id("DiffSuppressFunc")] = jen.Qual(SchemaUtilPackage, "IPFilterArrayDiffSuppressFunc")
```

Comment:
not sure, but I believe we already had this changed

Reply:
This is what is happening in the master 
```
➜  terraform-provider-aiven git:(main) go generate
ok      github.com/aiven/terraform-provider-aiven/internal/schemautil/userconfig        0.073s
➜  terraform-provider-aiven git:(main) ✗ git status
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   internal/schemautil/userconfig/dist/service_types.go

no changes added to commit (use "git add" and/or "git commit -a")
```
And diff :
```
@@ -64,9 +64,9 @@ func ServiceTypeCassandra() *schema.Schema {
                },
                "ip_filter": {
                        Description:      "Allow incoming connections from CIDR address block, e.g. '10.20.0.0/16'",
-                       DiffSuppressFunc: schemautil.IPFilterArrayDiffSuppressFunc,
+                       DiffSuppressFunc: schemautil.IpFilterArrayDiffSuppressFunc,
                        Elem: &schema.Schema{
-                               DiffSuppressFunc: schemautil.IPFilterValueDiffSuppressFunc,
+                               DiffSuppressFunc: schemautil.IpFilterValueDiffSuppressFunc,
                                Type:             schema.TypeString,
                        },
```

Reply:
So I guess we have to fix that

Reply:
ahh sorry, this is fixed in v4 branch